web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1293,12 +1323,11 @@ $.extend(fixmystreet.set_up, {
 
     if ($('.mobile').length) {
         // Make sure we end up with one Get updates link
-        if ($('#key-tools a.js-feed').length) {
+        if ($('#key-tools-mobile a.js-feed').length) {
```

Comment:
This change shouldn't be made - the js-feed is still in key-tools on any page it is used (not a report page), and needs to moved from there to sub_map_links.
And I'm afraid that sub_map_links uses a background image, so you then end up with two RSS feed icons being shown.
<img width="514" alt="image" src="https://github.com/user-attachments/assets/7ba47f55-354e-4034-ab17-0ba53206c3f2">